### PR TITLE
Linux ftw

### PR DESCRIPTION
**Pull request by leonthoma**

### CODE REVIEW

File: task2/task2.Rmd [hunk 1]
```diff
@@ -58,6 +58,26 @@ This task is considered to be finished when the chunk below is producing the out
 SELECT * FROM overview
 ```
 
+## hobos within 2km radius query
+
+```{sql connection=con}
+SELECT hobo_id,
+  st_astext(st_buffer(st_transform(geom, 25832), 2000)) as "hobos_within_2km"
```
Comment (mmaelicke) [inline task2/task2.Rmd]: Good starting point! Be careful as your queries are only buffering the geometry of each hobo. Finally, you want to obtain the hobos **within** the buffers you just created.

Comment (leonthoma) [reply task2/task2.Rmd]: I tried to use something like this
```sql
`SELECT 
    count(DISTINCT hobo_id FROM hobo WHERE st_within(st_transform(geom, 25832), 
    st_buffer((SELECT st_transform(geom, 25832) FROM hobo), 2000))) as hobos_within_2km_2020
FROM hobos WHERE year = 2020`
```
But that only works when one tuple is selected e.g. WHERE id = 1
Is it possible to loop through all the values of the hobo_id col ?

Comment (mmaelicke) [reply task2/task2.Rmd]: Looping in a sense like used in R is not really necessary. You can replace any value by an attribute name and as the query will be applied to the whole table row-wise anyway, the attribute name will be *replaced* by the value for that specific row.

I think what you want to do here, is to apply a filter condition to the nested query (the count part) that selects all **other** hobos within the buffer around the *current* geom attribute?

Maybe this helps. If you are still stuck, comment again, maybe with the current code, and I can further help you.

**Review by gremax93 — COMMENTED** [review]
Nice Job so far!

**Comment by MarcDjahan** [comment]
Nice work! Seems like all is working. I didnt get distance code so far but I will look in it later.

**Comment by FFHYDRO** [comment]
great job!

Am Mo., 20. Jan. 2020 um 10:36 Uhr schrieb MarcDjahan <
notifications@github.com>:

**Comment by FFHYDRO** [comment]
great job!

Am Fr., 17. Jan. 2020 um 15:49 Uhr schrieb leonthoma <
notifications@github.com>:

**Pull request closed without merging**